pkg/test/env/istio.go

```diff
@@ -59,6 +59,9 @@ var (
 	// nolint: golint
 	PULL_POLICY Variable = "PULL_POLICY"
 
+	// VERSION is Docker tag used for the base image.
+	VERSION Variable = "READ_FROM_ENVIRONMENT"
```

Comment:
What is `READ_FROM_ENVIRONMENT`?

Reply:
this could be better named.  Basically it is meant to indciate an error that the environment variable is not properly set.  We can probably do a little better by defaulting.

I am just not keen to keep multiple copies of the version hidden in the release...

Reply:
Sorry I am not understanding.. Later you call `Version:    env.VERSION.Value(),` which will set `Version: os.Getenv("READ_FROM_ENVIRONMENT")`. If we don't want to actually read the VERSION environment variable, then can we just get rid of this? And if we do want to actually read the env variable, it should be `VERSION Variable = "VERSION"`?

Reply:
I see thanks.

Reply:
addressed in latest pr.